tests/PythonQtTestMain.cpp

```diff
@@ -52,15 +52,15 @@ int main( int argc, char **argv )
 
   int failCount = 0;
   PythonQtTestApi api;
-  failCount += QTest::qExec(&api, argc, argv);
+  failCount |= QTest::qExec(&api, argc, argv);
   PythonQtTestSignalHandler signalHandler;
-  failCount += QTest::qExec(&signalHandler, argc, argv);
+  failCount |= QTest::qExec(&signalHandler, argc, argv);
   PythonQtTestSlotCalling slotCalling;
-  failCount += QTest::qExec(&slotCalling, argc, argv);
+  failCount |= QTest::qExec(&slotCalling, argc, argv);
 
   PythonQt::cleanup();
 
-  if (failCount>0) {
+  if (failCount) {
```

Comment:
Firstly, there are only two possible values for `main` to return: EXIT_SUCCESS and EXIT_FAILURE. Probably, nowadays, we can use 0 and 1 for this, but it is more about POSIX, rather than C++.  And the proposed pattern from docs use OR instead of addition too.
Secondly, I cannot remember the origin of this change :))), but seems like it was the problem here. Maybe, it was just some sanitizer complaining...

Reply:
It's not about the return value (that would be EXIT_FAILURE anyway, but about the logging. As I wrote, not that helpful, but it adds a bit of information...

Reply:
Ooops... Agree :) I've broken the error logging. Mistake.